emulator.md

```diff
@@ -96,21 +92,22 @@ From the shell, type and run
   # You may want to add these variables to your bashrc file.
 
   # If you are using a standalone repo, the path probably will be:
-  #     $HOME/freeflyer_android
+  #     $HOME/astrobee_android
   # If you are using this repo as a submodule, the path may be:
-  #     $HOME/freeflyer/submodules/android
+  #     $HOME/astrobee/src/submodules/android
   export ANDROID_PATH="insert here the path to android repository"
 
   # Location of emulator executable file. You may have a different path depending
-  # on your installation process.
+  # on your installation process. Also depending on your version of Android Studio,
+  # instead of tools/emulator it may be emulator/emulator
   export EMULATOR=$HOME/Android/Sdk/tools/emulator
```

Comment:
Can we bring the list-avds step after this? so we can use the EMULATOR variable instead of a path (since it may be different from user to user)

Reply:
Done.